### DISCUSSION TITLE
Comparing tour durations

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1094] [General]
Hello
How can I compare tours on duration of each of them? (no need of altitude tour comparison since they are identical)

**Comment by wolfgang-ch** [comment]
Can you create a screenshot where it is visible what you are missing.

**Comment by gringojack** [comment]
Hi
I m not missing on anything. I just need to know the process for How I can compare tours on duration of each of them? (no need of altitude tour comparison since they  are all identical)

**Comment by wolfgang-ch** [comment]
I'm lost, I don't understand your question, it does not help when you repeat your first question
